oa-enterprise/lib/omniauth/strategies/cas.rb

```diff
@@ -8,6 +8,29 @@ class CAS
       autoload :Configuration, 'omniauth/strategies/cas/configuration'
       autoload :ServiceTicketValidator, 'omniauth/strategies/cas/service_ticket_validator'
 
+      attr_accessor :raw_info
+      alias_method :user_info, :raw_info
+      
+      option :uid_key,              'user'
```

Comment:
While the `option` method is available in [modern implementations of `OmniAuth::Strategy`](https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategy.rb#L74), it is not available in the **github/omniauth** [version](https://github.com/github/omniauth/blob/cas-protocol-3.0-attribute-support/oa-core/lib/omniauth/strategy.rb#L3)
Need to revert back to a regular `attr_accessor` with a default.